.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: c
 
+cache:
+  pip: true
+  directories:
+    - cmocka
```

Comment:
This is redundant now since it is installed as a package, right?

Reply:
Yes, this should have been dropped and I missed that.